worlds/ladx/test/__init__.py

```diff
@@ -0,0 +1,4 @@
+from test.TestBase import WorldTestBase
+from ..Common import LINKS_AWAKENING
+class LADXTestBase(WorldTestBase):
+    game = LINKS_AWAKENING
```

Comment:
Want to fix that missing newline at end of file?

Reply:
no

Reply:
did it anyhow tho